openwisp_utils/ci.py

```diff
@@ -0,0 +1,30 @@
+"""
+This file contains functions that are required by
+multiple openwisp repositories for testing
+during continuous integration
+"""
+import os
+import re
+
+
+def check_migration_name(migration_path, migrations_to_ignore):
+    """
+    Check if the migration files created have a descriptive
+    name; if default name pattern is found, exit with 1
+    Arguements:
+      - migration_path: absolute path to `migrations/` folder
+      - migrations_to_ignore: number of migrations after
+        which checking should begin, say, if checking needs to
+        start after `0003_auto_20150410_3242.py` value will be `3`
+    """
+    migrations_to_ignore = migrations_to_ignore
```

Comment:
Why this line @atb00ker

Reply:
Oops, it was unintentional. Thanks! :blush: